httpapi/server.go

```diff
@@ -0,0 +1,26 @@
+package httpapi
+
+import (
+	"net/http"
+	"time"
+
+	"github.com/julienschmidt/httprouter"
+	"github.com/serverless/event-gateway-connector/connection"
+)
+
+// StartConfigAPI creates a new configuration API server and listens for requests.
+func StartConfigAPI(store connection.Service) {
```

Comment:
I would return `error` from here, that way we can return `Fatal` from the main caller.

Reply:
yup